vars/neoDeploy.groovy

```diff
@@ -27,14 +27,17 @@ def call(parameters = [:]) {
         'warAction'
         ]
 
+    List generalConfigurationKeys = [
+        'neoHome'
+        ]
+
     List stepConfigurationKeys = [
         'account',
         'dockerEnvVars',
         'dockerImage',
         'dockerOptions',
         'host',
-        'neoCredentialsId',
-        'neoHome'
```

Comment:
Shouldn't it be possible to overwrite the default via step configuration.
Beside the fact that it is reasonable to provide the default setting on step level, we are less backward compatible with removing this by the way.